test/OrchardCore.Tests/OrchardCore.Tests.csproj

```diff
@@ -80,6 +80,12 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Localization\*\OrchardCore.Tests.po">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
```

Comment:
No need, you can add them as embedded resources

Reply:
They are not same.

Reply:
Why do we need to preserve the newest?

Reply:
Because the target rule needs to copy it to the output directory.